python/orca/src/bigdl/orca/learn/pytorch/core/lifecycle.py

```diff
@@ -28,9 +28,12 @@ def setup_torch_distribute(self, tcp_store_host, tcp_store_port, world_rank,
                                world_size):
         self._init_torch_ddp(tcp_store_host, tcp_store_port, world_rank,
                              world_size)
+        self.setup_ddp_components()
 
+    def setup_torch_estimator(self, world_rank, world_size):
+        self.rank = world_rank
+        self.size = world_size
```

Comment:
remove these two lines in _init_torch_ddp?
setup_torch_estimator would always be called before setup_torch_distribute right?